### PR TITLE
Add icons to key features

### PR DESCRIPTION
**Pull request by angelikatyborska**
Closes https://github.com/exercism/elixir/issues/565

The list of icons was defined recently: https://github.com/exercism/docs/commit/8998f31d247fd87317af5cd428bad934513eb01e

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -3136,32 +3136,32 @@
...
       "title": "Dynamically typed",
       "content": "Elixir has no compile-time type checks, favoring run-time pattern matching.",
-      "icon": "TODO"
+      "icon": "dynamically-typed"
     },
     {
       "title": "Immutable",
       "content": "All data in Elixir is immutable, allowing for safer and easier-to-reason-about concurrency.",
-      "icon": "TODO"
+      "icon": "easy"
```
Comment (angelikatyborska) [inline config.json:3154]: I think that one is a bit of a stretch, but I don't see any icon that would fit well.

Comment (angelikatyborska) [reply config.json:3154]: that's the icon: https://d31gpyts0tlbet.cloudfront.net/key-features/easy.svg does that fit?

Comment (angelikatyborska) [reply config.json:3154]: Alternatively:  https://d31gpyts0tlbet.cloudfront.net/key-features/homoiconic.svg

**Comment by ee7** [comment]
FYI: this PR beat the new configlet release by 2 minutes. `configlet lint` will check the icon values on any further pushes.

**Comment by ee7** [comment]
FYI: this PR beat the new configlet release by 2 minutes. `configlet lint` will check the icon values on any further pushes.

File: config.json (hunk 1 above)
Comment (ee7) [inline config.json:3154]:
```suggestion
      "icon": "immutable"
```

Now available if you want it.

**Review by ee7 — APPROVED** [review]

**Review by neenjaw — APPROVED** [review]

**Pull request merged**